internal_transfer_with_agreed_amount/__manifest__.py

```diff
@@ -1,20 +1,20 @@
 {
-    'name': 'Internal transfers with an agreed amount',
```

Comment:
Commit title:

Take into account text after module name should be lowercased, i.e.:

`[IMP] module_name: black, isort, prettier`

Check all other commits

Reply:
Changes applied in all 8 commits.